src/django_components/context.py

```diff
@@ -38,77 +81,82 @@ def set_slot_fill(context: Context, component_id: str, slot_name: str, value: "F
 
     Note that we make use of the fact that Django's Context is a stack - we can push and pop
     extra contexts on top others.
-
-    For the slot fills to be pushed/popped wth stack layer, they need to have keys defined
-    directly on the Context object.
     """
     trace_msg("SET", "FILL", slot_name, component_id)
-    slot_key = (_FILLED_SLOTS_CONTENT_CONTEXT_KEY, component_id, slot_name)
-    context[slot_key] = value
+    slot_key = f"{component_id}__{slot_name}"
```

Comment:
Changed the keys from tuples to strings so the data can be serialized to JSON, which I used during debugging